[2024-05-21T14:36:27.314+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-21T14:36:27.331+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: statistical_calculations_dag.generate_random_numbers scheduled__2022-01-01T00:00:00+00:00 [queued]>
[2024-05-21T14:36:27.338+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: statistical_calculations_dag.generate_random_numbers scheduled__2022-01-01T00:00:00+00:00 [queued]>
[2024-05-21T14:36:27.339+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 6
[2024-05-21T14:36:27.391+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): generate_random_numbers> on 2022-01-01 00:00:00+00:00
[2024-05-21T14:36:27.397+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=6166) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-05-21T14:36:27.398+0000] {standard_task_runner.py:63} INFO - Started process 6170 to run task
[2024-05-21T14:36:27.398+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'statistical_calculations_dag', 'generate_random_numbers', 'scheduled__2022-01-01T00:00:00+00:00', '--job-id', '1766', '--raw', '--subdir', 'DAGS_FOLDER/second_dag.py', '--cfg-path', '/tmp/tmp6cc8yqha']
[2024-05-21T14:36:27.399+0000] {standard_task_runner.py:91} INFO - Job 1766: Subtask generate_random_numbers
[2024-05-21T14:36:27.441+0000] {task_command.py:426} INFO - Running <TaskInstance: statistical_calculations_dag.generate_random_numbers scheduled__2022-01-01T00:00:00+00:00 [running]> on host c420abfdc210
[2024-05-21T14:36:27.527+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='test' AIRFLOW_CTX_DAG_ID='statistical_calculations_dag' AIRFLOW_CTX_TASK_ID='generate_random_numbers' AIRFLOW_CTX_EXECUTION_DATE='2022-01-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2022-01-01T00:00:00+00:00'
[2024-05-21T14:36:27.528+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-21T14:36:27.555+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-05-21T14:36:27.557+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-21T14:36:27.564+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=statistical_calculations_dag, task_id=generate_random_numbers, run_id=scheduled__2022-01-01T00:00:00+00:00, execution_date=20220101T000000, start_date=20240521T143627, end_date=20240521T143627
[2024-05-21T14:36:27.612+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-21T14:36:27.621+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
